Gemfile.lock

```diff
@@ -2,6 +2,7 @@ PATH
   remote: .
   specs:
     routemaster-drain (2.4.4)
+      addressable
```

Comment:
I think runtime dependencies are taken account in that block by `routemaster-drain!` - i.e. none of the other gemspec dependencies are in that block, just development dependencies.